the-collab-lab/tcl-22-smart-shopping-list
# Style Welcome page

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



Add styling to (desktop version of) Welcome page to match [Figma mocks](https://www.figma.com/file/VH8Z8iouh7F9zBBv1kbuq5/Shopping-App-design?node-id=106%3A105&viewport=846%2C-10930%2C1.5277392864227295).

Mobile view is still a work-in-progress.

Have not yet been able to successfully hide the top green bar w/logo only while on this page. Would also love to animate the transition between the two different views. Any insight into either of these issues would be very welcome! 😊 


## Acceptance Criteria

From Figma mocks:
![image](https://user-images.githubusercontent.com/58532369/120056219-d920aa00-c008-11eb-98d2-da09745d79c0.png)
![image](https://user-images.githubusercontent.com/58532369/120056225-e2117b80-c008-11eb-97f1-f08e953ec273.png)



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |
|  ✓ | :nail_care: Style              |

## Updates

### Before
![image](https://user-images.githubusercontent.com/58532369/120055761-16d00380-c006-11eb-9ed0-a2481761d06a.png)
![image](https://user-images.githubusercontent.com/58532369/120055758-120b4f80-c006-11eb-96c5-3e2c3872d780.png)


### After
![image](https://user-images.githubusercontent.com/58532369/120056305-3fa5c800-c009-11eb-89c0-1eeffb66b704.png)

![image](https://user-images.githubusercontent.com/58532369/120071127-c6839080-c05b-11eb-9985-c9e48181ee95.png)



## Testing Steps / QA Criteria



1. 👀
2. 👍  ❓  🎉   : 🔨

(make sure to clear tokens from local state if they're there, or open in an incognito window in order to open to the Welcome screen)
